### PR TITLE
Renamed creatby and group_id to camel case variants.

### PR DESCRIPTION
**Pull request by l-e-u**
# What does this PR do? 
Reviews the model schemas to ensure the properties are typed in camel case which resolves issue #47 .

# How does this PR accomplish that? 
Renamed any properties that were not in camel case.

### CODE REVIEW

File: src/models/Tasks.js [hunk 1]
```diff
@@ -1,23 +1,23 @@
...
-    points: {
-        type: String,
-        required: false,
-    },
-})
+  name: {
+    type: String,
+    required: true,
+  },
+  createdBy: {
```
Comment (MarchandMD) [inline src/models/Tasks.js:9]: - updated here
- Swagger documentation will still need to be updated
- @jfarelli if you're calling `.createdby` anywhere in the FE, it'll need to be updated to `.createdBy`

File: src/models/Players.js [hunk 2]
```diff
@@ -1,27 +1,27 @@
...
+  score: {
+    type: String,
+    required: false,
+  },
+  email: {
+    type: String,
+    required: false,
+    default: "email@email.com",
+  },
+  groupId: {
```
Comment (MarchandMD) [inline src/models/Players.js:21]: - this'll need to be updated in Swagger documentation, too. I'll handle this
- @jfarelli this also changed; so heads up if it's being used in the front end

**Review by MarchandMD — COMMENTED** [review]
@msakr21 Remind me about this next time we pair program

**Review by MarchandMD — APPROVED** [review]
LGTM

**Comment by MarchandMD** [comment]
@l-e-u Hey thanks for contributing! 

Just out of curiousity, how did you find our humble little project?

**Comment by l-e-u** [comment]
@MarchandMD 
👍 I'm glad I could help, as small as it was lol.

I'm a junior that's been working solo on projects, and want more experience reading others' code, plus developing with others helps everyone out 😊.

I found this particular issue as a top result after using some of the filters on GitHub.

**Pull request merged**